### PR TITLE
Fix incorrect 'Link to Crossdart' links

### PR DESCRIPTION
**Pull request by astashov**
They were generated incorrectly (whoops!)
Fixed that.

### CODE REVIEW

**Comment by devoncarew** [comment]
@keertip @sethladd

**Comment by sethladd** [comment]
Looks like tests are failing because the version of analyzer is still using dart:profiler.

```
The built-in library 'dart:profiler' is not available on the stand-alone VM.
'package:analyzer/src/generated/utilities_general.dart': error: line 10 pos 1: library handler failed
import 'dart:profiler';
^
```

**Comment by sethladd** [comment]
The tests on stable are passing, so I think we can merge this in.

LGTM

File: lib/src/model.dart [hunk 1]
```diff
@@ -22,6 +22,7 @@ import 'package_meta.dart' show PackageMeta, FileContents;
 import 'cache.dart';
 
 import '../markdown_processor.dart' show Documentation, renderMarkdownToHtml;
+import 'package:analyzer/src/generated/source_io.dart';
```
Comment (sethladd) [inline lib/src/model.dart]: We shouldn't import a file from src. Is this absolutely necessary?

Comment (keertip) [inline lib/src/model.dart]: We can make an exception for the analyzer package. It has public api in `src`.

Comment (sethladd) [inline lib/src/model.dart]: hah. well, ok then :)

**Comment by sethladd** [comment]
deferring review to @keertip

**Comment by sethladd** [comment]
Once we address https://github.com/dart-lang/dartdoc/issues/972 then this PR should pass its Travis builds.

**Comment by keertip** [comment]
restarting build

**Comment by keertip** [comment]
lgtm!

**Pull request merged**